bootstrap.sh

```diff
@@ -66,23 +66,42 @@ fi
 git submodule update --init --recursive
 
 kernel_ver=`uname -r`
+kernel_hed=`sudo apt search linux-headers-$(uname -r)`
+
 echo "Running kernel version: $kernel_ver"
+echo "..................................."
+echo "Running kernel headers: $kernel_hed"
+
 mj_ver=$(echo $kernel_ver | cut -d. -f1)
 mn_ver=$(echo $kernel_ver | cut -d. -f2)
+mj_hed=$(echo $kernel_hed | cut -d. -f7 | cut -d- -f3)
+mn_hed=$(echo $kernel_hed | cut -d. -f8)
 
-if [[ "$mj_ver" < "5" ]] || [[ "$mn_ver" < "6" ]]; then
-        tput setaf 1
-	echo "Mizar requires an updated kernel: linux-5.6-rc2 for TCP to function correctly. Current version is $kernel_ver"
-
-	read -p "Execute kernel update script (y/n)?" choice
-	tput sgr0
-	case "$choice" in
-	  y|Y ) sh ./kernelupdate.sh;;
-	  n|N ) echo "Please run kernelupdate.sh to download and update the kernel!"; exit;;
-	  * ) echo "Please run kernelupdate.sh to download and update the kernel!"; exit;;
-	esac
+if ([[ "0$mj_ver" -le "04" ]] || [[ "$mn_ver" -le "05" ]]) || ([[ "$mn_ver" -le "5" ]]); then
```

Comment:
What is "0$mj_ver"? And why do we need this change?

Reply:
Previously the condition was not getting satisfied without the concatenation of '0', but now, we have optimized the condition's logic and the zero is removed and the code is updated.